Lesson1/HomeWork1.py

```diff
@@ -0,0 +1,21 @@
+# we have norway text in old style formatting
+# re-write the same text as:
+# #1 string with format() call
+# #2 f-string
+# use linter(https://github.com/wemake-services/wemake-python-styleguide)
+# to check your new created python module for possible linter errors
+# try to run code from pycharm/command line
+
+norway_text = 'Automatisering akselererer %syeblikket da roboter vil ' \
```

Comment:
Not critical but I recommend to use smth aka 
```
text = ('abcd'
        'fegk')
```
Such construction without slashes more readable + will solve linter N400 warnings